integration-tests/README.md

```diff
@@ -74,41 +66,32 @@ npm test
 ```
 
 If you're iterating on the tests to solve an issue which shows itself in a particular environment, the workflow could be:
-1. Rebuild the Realm JS native module (if you're making changes to the C++ code), by running `npm run build-changes` in
-   in the project root.
+1. Rebuild the Realm JS native module (if you're making changes to the C++ code), by running `npm run build-changes` in the project root.
 2. Produce a packaged version of Realm JS, by running `npm run realm/pack` in the `./integration-tests` directory.
-3. Re-install the packaged version of Realm JS, by running `npm install` in the environments directory. This will
-   install Realm JS from the packaged version you've just created.
+3. Re-install the packaged version of Realm JS, by running `npm install` in the environments directory. This will install Realm JS from the packaged version you've just created.
 
 ### How to write tests
 
-Because of limitiations (see below), we need to explicit require in the files which defines the individual tests: To
-write a new test, simply add it in the relevant file in `test/src/` or create a new file and make sure to require
-that from `test/src/index.js`.
+Because of limitiations (see below), we need to explicit require in the files which defines the individual tests: To write a new test, simply add it in the relevant file in `test/src/` or create a new file and make sure to require that from `test/src/index.ts`.
 
 Tests will have access to the following globals:
 
 - [the Mocha hook globals](https://mochajs.org/#hooks) (define, it, after, before, etc.).
 - `Realm` the Realm constructor.
-- `fs` the lowest common denominator of the [`fs-extra`](https://www.npmjs.com/package/fs-extra) and
-  [`react-native-fs`](https://www.npmjs.com/package/react-native-fs) APIs.
-- `path` the lowest common denominator of the Node.js path interface and a
-  [node-independent implementation of Node's path](https://www.npmjs.com/package/path-browserify) module.
+- An `UndocumentedRealm` which extends the `Realm` constructor with some methods which are not documented or considered a part of the public Realm JS API.
```

Comment:
While there are undocumented methods/properties, we should aim to have types for all methods, including the private ones.

Reply:
I don't disagree - but I don't know if we have consensus on that: Would @kneth or @bmunkholm object against that?

Reply:
It's fine with me. You are basically using `UndocumentedRealm` as a box with the label "Nothing important to see".